core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/RawlsApiService.scala

```diff
@@ -65,7 +66,16 @@ trait RawlsApiService //(val workspaceServiceConstructor: UserInfo => WorkspaceS
 
   def apiRoutes = options { complete(OK) } ~ workspaceRoutes ~ entityRoutes ~ methodConfigRoutes ~ submissionRoutes ~ adminRoutes ~ userRoutes ~ billingRoutes ~ notificationsRoutes ~ servicePerimeterRoutes
 
-  def route: server.Route = (logRequestResult & handleExceptions(RawlsApiService.exceptionHandler)) {
+  implicit def rejectionHandler = {
+    import spray.json._
+    import DefaultJsonProtocol._
+    RejectionHandler.default.mapRejectionResponse {
+      case res @ HttpResponse(status, _, ent: HttpEntity.Strict, _) =>
+        res.copy(entity = HttpEntity(ContentTypes.`application/json`, Map(status.toString -> ent.data.utf8String).toJson.toString))
```

Comment:
what value does `status.toString` return? Maybe use `status.value` here to offer the end user the most info?

Reply:
in my test it returned `400 Bad Request` which i thought was pretty good

Reply:
in my test it returned `400 Bad Request` which i thought was pretty good

Reply:
which is exactly what  `.value` returns - all  good then.